src/pretix/control/datasync.py

```diff
@@ -0,0 +1,84 @@
+#
+# This file is part of pretix (Community Edition).
+#
+# Copyright (C) 2014-2020 Raphael Michel and contributors
+# Copyright (C) 2020-2021 rami.io GmbH and contributors
+#
+# This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General
+# Public License as published by the Free Software Foundation in version 3 of the License.
+#
+# ADDITIONAL TERMS APPLY: Pursuant to Section 7 of the GNU Affero General Public License, additional terms are
+# applicable granting you additional permissions and placing additional restrictions on your usage of this software.
+# Please refer to the pretix LICENSE file to obtain the full terms applicable to this work. If you did not receive
+# this file, see <https://pretix.eu/about/en/license>.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied
+# warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Affero General Public License along with this program.  If not, see
+# <https://www.gnu.org/licenses/>.
+#
+
+from itertools import groupby
+
+from django.contrib import messages
+from django.dispatch import receiver
+from django.http import HttpResponseNotAllowed
+from django.shortcuts import redirect
+from django.template.loader import get_template
+from django.utils.translation import gettext_lazy as _
+
+from pretix.base.datasync.datasync import sync_targets
+from pretix.base.models import Event, Order
+from pretix.control.signals import order_info
+from pretix.control.views.orders import OrderView
+
+
+@receiver(order_info, dispatch_uid="datasync_control_order_info")
```

Comment:
I don't think we use order_info internally elsewhere, would we want to wire this into the OrderView instead?

Reply:
I think we should start using it (and similar) more internally. It's helpful to have more modular code separated by feature groups like "data sync". 

Also orders/views.py and pretixcontrol/order/index.html are long enough already.
I like the structure better if we keep the template for this block in a separate file (pretixcontrol/datasync/control_order_info.html), and the template vars separated in the signal receiver. We could of course directly {% include %} that file from order/index.html, but then we'd place template vars in the larger context of OrderDetail and pass it through order/index.html.